mmv1/products/developerconnect/Connection.yaml

```diff
@@ -12,205 +12,381 @@
-            min_version: 'beta'
+            description: |-
+              Required. A SecretManager resource containing the OAuth token that authorizes
+              the connection. Format: `projects/*/secrets/*/versions/*`.
             required: true
-          - name: 'username'
+          - name: username
             type: String
-            description: |
-              Output only. The username associated with this token.
-            min_version: 'beta'
+            description: Output only. The username associated with this token.
             output: true
-      - name: 'appInstallationId'
+      - name: appInstallationId
         type: String
-        description: |
-          Optional. GitHub App installation id.
-        min_version: 'beta'
-      - name: 'installationUri'
+        description: Optional. GitHub App installation id.
+  - name: githubEnterpriseConfig
+    type: NestedObject
+    description: Configuration for connections to an instance of GitHub Enterprise.
+    properties:
+      - name: appSlug
         type: String
-        description: |
-          Output only. The URI to navigate to in order to manage the installation
-          associated with this GitHubConfig.
-        min_version: 'beta'
+        description: Output only. The URL-friendly name of the GitHub App.
         output: true
-  - name: 'name'
+      - name: privateKeySecretVersion
+        type: String
+        description: |-
+          Optional. SecretManager resource containing the private key of the GitHub App,
+          formatted as `projects/*/secrets/*/versions/*`.
+      - name: installationUri
+        type: String
+        description: |-
+          Output only. The URI to navigate to in order to manage the installation associated
+          with this GitHubEnterpriseConfig.
+        output: true
+      - name: serviceDirectoryConfig
+        type: NestedObject
+        description: |-
+          ServiceDirectoryConfig represents Service Directory configuration for a
+          connection.
+        properties:
+          - name: service
+            type: String
+            description: |-
+              Required. The Service Directory service name.
+              Format:
+              projects/{project}/locations/{location}/namespaces/{namespace}/services/{service}.
+            required: true
+      - name: serverVersion
+        type: String
+        description: Output only. GitHub Enterprise version installed at the host_uri.
+        output: true
+      - name: sslCaCertificate
+        type: String
+        description: Optional. SSL certificate to use for requests to GitHub Enterprise.
+      - name: hostUri
+        type: String
+        description: Required. The URI of the GitHub Enterprise host this connection is for.
+        required: true
+      - name: appId
+        type: String
+        description: Optional. ID of the GitHub App created from the manifest.
+      - name: webhookSecretSecretVersion
```

Comment:
Can you confirm if  this field is immutable or not? `immutable: true` is added into other `webhookSecretSecretVersion` fields.